drivers/sensor/sun8i-thermal.c

```diff
@@ -21,49 +21,53 @@
 
 #define THS_FILTER_CONTROL_REG 0x70
 
+#if CONFIG_SOC_H5
+#define HAVE_THS_CPU1
+#endif
+
 struct sun8i_thermal_sensor_info {
 	struct   sensor_info info;
 	const uint16_t       address;
 };
 
 enum {
-	THS0,
-	THS1,
-	THS2,
-	SENSOR_COUNT
+	THS_CPU0,
```

Comment:
This goes against the naming convention found in both the A64 and the H5 documentation